docs/supported_op_devices.py

```diff
@@ -1,5 +1,6 @@
 from nvidia.dali import backend as b
 import nvidia.dali.ops as ops
+import nvidia.dali.plugin.pytorch
```

Comment:
do you need that?

Reply:
Yes, because the operator is registered during the module initialization.